.github/workflows/tests-vmtests.yml

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Tests VMTests suite
+
+permissions:
+  contents: read
+  # Azure login.
+  id-token: write
+
+on:
+  workflow_call:
+    inputs:
+      hostArch:
+        required: true
+        type: string
+      hostDistro:
+        required: true
+        type: string
+
+env:
+  EXPECTED_GO_VERSION: "1.24.1"
+  ORAS_VERSION: "1.2.3"
+
+jobs:
+  tests-vmtests:
+    name: Tests VMTests suite
+    runs-on:
+    - self-hosted
+    - 1ES.Pool=${{ inputs.hostDistro == 'azl3' && (inputs.hostArch == 'amd64' && 'maritimus-github-runner-azl3-amd64' || 'maritimus-github-runner-azl3-arm64') || 'maritimus-github-runner-ubuntu2404-amd64'}}
+    permissions:
+      contents: read
+      # Azure login.
+      id-token: write
+    environment: public
+    steps:
+    - name: setup go 1.x
+      uses: actions/setup-go@v5
+      with:
+        go-version: "${{ env.EXPECTED_GO_VERSION }}"
+
+    - name: Install prerequisites (AZL3)
+      if: inputs.hostDistro == 'azl3'
+      run: |
+        set -eux
+
+        sudo tdnf install -y libvirt libvirt-daemon libvirt-daemon-config-network \
+          libvirt-daemon-kvm libvirt-devel qemu-kvm qemu-img python3-libvirt \
+          python3-devel edk2-ovmf \
+          azure-cli
+
+        sudo tdnf list installed
+
+        sudo systemctl restart libvirtd
+        sudo systemctl status libvirtd
+
+    - name: Install prerequisites (Ubuntu 24.04)
+      if: inputs.hostDistro == 'ubuntu2404'
+      run: |
+        set -eux
+
+        sudo apt update -y
+        sudo apt -y install python3.12-venv python3-pip python3-dev \
+            libvirt-dev libvirt-daemon libvirt-daemon-system libvirt-clients \
+            qemu-kvm virt-manager
+
+        # Install arm64 specific
+        if [[ "$HOST_ARCH" == "arm64" ]]; then
+          sudo apt -y install qemu-system-arm qemu-efi-aarch64 ovmf vgabios
+        fi
+
+        sudo apt list --installed
+
+        curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+      env:
+        HOST_ARCH: ${{ inputs.hostArch }}
```

Comment:
just curious, why `HOST_ARCH` is set as an env var with `hostArch` value,  but `hostDistro` is used directly?